### PR TITLE
Task dev 2

### PR DESCRIPTION
**Pull request by LykashevichAndrei**

### CODE REVIEW

File: task_DEV-2/EvenElementsOfTheString.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using System.Collections.Generic;
+using System.Text;
+namespace task_DEV_2
+{
+    /// <summary>
+    /// This class finds the sequence of even indices.
+    /// </summary>
+    public class EvenElementsOfTheString
+    {
+       private string str;
```
Comment (IgorKheidorov) [inline task_DEV-2/EvenElementsOfTheString.cs]: str bad name

File: task_DEV-2/EvenElementsOfTheString.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline task_DEV-2/EvenElementsOfTheString.cs]: Collections? I do not see here collections

File: task_DEV-2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using System.Text;
+namespace task_DEV_2
+{ 
+  /// <summary>
+  /// The entry point to the program.
+  /// </summary>
+    class EnteryPoint
+    {
+        static void Main()
+        {
```
Comment (IgorKheidorov) [inline task_DEV-2/Program.cs]: Code style

File: task_DEV-2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+                EvenElementsOfTheString s = new EvenElementsOfTheString(); //create an instance of the class
```
Comment (IgorKheidorov) [inline task_DEV-2/Program.cs]: s very bad name

File: task_DEV-2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+namespace task_DEV_2
+{ 
+     /// <summary>
+     /// The entry point to the program.
+     /// </summary>
+    class EnteryPoint
+    {
+        static void Main()
+        {
+                EvenElementsOfTheString mystring = new EvenElementsOfTheString(); //create an instance of the class
```
Comment (AlexanderSoroka) [inline task_DEV-2/Program.cs]: remove useless comments

Comment (AlexanderSoroka) [inline task_DEV-2/Program.cs]: indentation is wrong

File: task_DEV-2/EvenElementsOfTheString.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    /// <summary>
+    /// This class finds the sequence of even indices.
+    /// </summary>
+    public class EvenElementsOfTheString
+    {
+       private string enteredstring;
+
+      public EvenElementsOfTheString() //constructor
+        {
+            this.enteredstring = Console.ReadLine();
```
Comment (AlexanderSoroka) [inline task_DEV-2/EvenElementsOfTheString.cs]: Single responsibility violation

File: task_DEV-2/EvenElementsOfTheString.cs [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+{
+    /// <summary>
+    /// This class finds the sequence of even indices.
+    /// </summary>
+    public class EvenElementsOfTheString
+    {
+       private string enteredstring;
+
+      public EvenElementsOfTheString() //constructor
+        {
```
Comment (AlexanderSoroka) [inline task_DEV-2/EvenElementsOfTheString.cs]: codestyle violation

File: task_DEV-2/EvenElementsOfTheString.cs [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+namespace task_DEV_2
+{
+    /// <summary>
+    /// This class finds the sequence of even indices.
+    /// </summary>
+    public class EvenElementsOfTheString
+    {
+       private string enteredstring;
+
+      public EvenElementsOfTheString() //constructor
```
Comment (AlexanderSoroka) [inline task_DEV-2/EvenElementsOfTheString.cs]: useless comment

**Pull request merged**